spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.feature 'Post Show', type: :feature do
+  let(:user) do
+    User.create(name: 'Tom', photo: 'https://www.kasandbox.org/programming-images/avatars/leaf-blue.png',
+                bio: 'He is a good programmer')
+  end
+  let!(:post) { Post.create(author: user, title: "first post's title", text: 'first text') }
+  let!(:comment1) { Comment.create(author: user, post:, text: 'first comment') }
+  let!(:comment2) { Comment.create(author: user, post:, text: 'second comment') }
+  let!(:like1) { Like.create(user:, post:) }
+
+  before do
+    visit user_post_path(user, post)
+  end
+
+  scenario 'see the post details' do
+    expect(page).to have_content('by Tom')
+    expect(page).to have_content('Comments: 2')
+    expect(page).to have_content('Likes: 1')
+  end
+
```

Comment:
- [ ] You missed adding the below tests for the posts#show page, Please make sure to add them and they should pass without any errors.
  - Tests that assert you can see the post's title.

Reply:
Resolved

Reply:
This change has been done well